app/webpacker/src/stylesheets/_rules_of_origin.scss

```diff
@@ -47,3 +47,15 @@
     margin-bottom: 0;
   }
 }
+
+.tariff-markdown {
```

Comment:
Why here rather than in _markdown.scss? AFAIK this is global for all tariff-markdown images?

Reply:
Yeh, this the 'wrong' place for this I think - I'll take a look

Reply:
Tweaked to only impact MD docs in RoO wizard